pyaedt/application/Variables.py

```diff
@@ -958,11 +958,7 @@ def delete_variable(self, sVarName):
 
 
 class Variable(object):
-    """Stores variables and provides for performing operations on variables.
-    
-    This class stores variables and provides for performing operations 
-    on variables. It handles the contents of design properties and project 
-    variables.
+    """Stores design properties and project variables and provides for performing operations on them.
```

Comment:
```suggestion
    """Stores design properties and project variables and provides operations to perform on them.
```
Reword? feel free to update my suggestion.